package.json

```diff
@@ -11,13 +11,13 @@
   "license": "MIT",
   "dependencies": {
     "async": "^2.3.0",
-    "disyuntor": "^2.0.0",
+    "disyuntor": "^3.4.2",
```

Comment:
We found several critical issues in this version of disyuntor, please update to 3.4.3

Reply:
Is there any vulnerable code in the version 2 of disyuntor? I'd advice not to update otherwise, 2.0.0 is really stable whereas 3 is not so battle tested and changes a lot + limitd is a service where each deployment involves downtime